routemaster/models/client_token.rb

```diff
@@ -0,0 +1,37 @@
+require 'routemaster/models'
+require 'routemaster/mixins/redis'
+require 'securerandom'
+
+module Routemaster
+  module Models
+    class ClientToken
+      include Mixins::Redis
+
+      PREFIX = "api_keys:"
+
+      def self.get_all
+        key_dump = {}
+        existing_keys = _redis.keys "#{PREFIX}*"
```

Comment:
I had came across that, but our use case here is fetching all keys at once, which we expect to be a relatively low number. I thought the point was moot, in that it's not quite the volume `select * from orders`. Additionally, it will very rarely need to be queried, so it essentially exists as a debugging interface